ui/src/lib/forms/feedingInformation.js

```diff
@@ -0,0 +1,82 @@
+import * as yup from 'yup';
+
+const feedingInfoFields = {
+  'Feeding Information': [
+    {
+      name: 'breastFeeding',
+      label: 'Breastfeeding:',
+      type: 'radio',
+      validate: yup.string().required('Breastfeeding status is required'),
+      width: {
+        xs: 12,
+        sm: 14,
+        md: 8,
+        lg: 24,
+      },
+      options: [{ label: 'Well', value: 'Well' }, { label: 'Poorly', value: 'Poorly' }, { label: 'Unable to breastfeed', value: 'Unable to breastfeed'}],
+    },
+    
+    {
+      name: 'otherFeedsBelowSixMonths',
+      label: 'Other feeds introduced below 6 months:',
+      type: 'text',
+      type: 'radio',
+      validate: yup.string().required('This field is required'),
```

Comment:
Avoid the display of generic error messages. Specify the field that is required or for which the error is being displayed. Example "Other feeds introduced below 6 months is required" rather than "This field is required". Change for all other affected fields.